zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/handlers/FormHandler.java

```diff
@@ -79,7 +81,13 @@ public void updateEntity(final Record<Form> record, final FormEntity entity) {
 
   @Override
   public void flush(final FormEntity entity, final BatchRequest batchRequest) {
-    batchRequest.add(indexName, entity);
+    final Map<String, Object> updateFields = new HashMap<>();
+    String id = entity.getId();
+    if (entity.getIsDeleted() != null && entity.getIsDeleted()) {
+      updateFields.put("isDeleted", true);
+      id = String.valueOf(entity.getKey());
+    }
```

Comment:
❌ Doesn't it mean we don't update the other properties then?

Reply:
For Forms specifically nothing gets updated apart from the `isDeleted` tag. Every form ingested has it's own key but carries the same `formId` and get's a version increment.